app/static/admin/css/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+@charset "UTF-8";/*!
```

Comment:
I prefer to load bootstrap css file using CDN (I highly discourage having it on disk)

Reply:
I'll update it. Thanks